cmd/mistry/main.go

```diff
@@ -67,29 +68,34 @@ JOB PARAMETERS:
    -- dynamic options for the command
 
 EXAMPLES:
-   1. The following sequence schedules a command to be executed in http://example.org:9090,
-      builds the yarn project, groups it under group_name, syncs the result to the
-      directory /tmp/yarn and uses the dynamic arguments yarn.lock and foo.
-      Note the @ before the yarn.lock, this indicates a referral to an actual file on the
-      filesystem.
+    1. Talk to the mistry service at http://example.org:9090, to build the yarn
+       project, group it under group_name, sync the result to the
+       directory /tmp/yarn and use the dynamic arguments yarn.lock and foo.
+       Note the @ before the yarn.lock, this indicates a referral to an actual file on the
+       filesystem.
 
-      $ mistry-cli build --host example.org --port 9090 --project yarn \
-      --group group_name --target /tmp/yarn -- --lockfile=@yarn.lock --foo=bar
+       $ {{.HelpName}} --host example.org --port 9090 --project yarn \
+	     --group group_name --transport rsync --target /tmp/yarn \
+	     -- --lockfile=@yarn.lock --foo=bar
 
-   2. The following sequence uses the GROUP environment variable to set the project group.
+    2. Uses the GROUP environment variable to set the project group.
```

Comment:
Since we're at it, I think this example isn't needed at all. A note that says that every flag can be provided as environment variable too should be sufficient.